### PR TITLE
Update OWNERS.

### PR DESCRIPTION
**Pull request by patrickdillon**
This should be everyone. 

/cc @abhinavdahiya

### CODE REVIEW

File: OWNERS_ALIASES [hunk 1]
```diff
@@ -8,11 +8,14 @@ aliases:
...
     - steveeJ
     - wking
+    - jhixson74
   installer-reviewers:
     - jstuever
     - mtnbikenc
     - patrickdillon
     - sdodson
+    - vrutkovs
+    - jcpowermac
```
Comment (abhinavdahiya) [inline OWNERS_ALIASES]: we keep this list sorted alphabetically

File: OWNERS_ALIASES [hunk 2]
```diff
@@ -8,11 +8,14 @@ aliases:
     - staebler
     - steveeJ
     - wking
+    - jhixson74
```
Comment (abhinavdahiya) [inline OWNERS_ALIASES]: its fine if we start @jhixson74 as reviewer like everybody else :)

**Comment by patrickdillon** [comment]
Failed with 
```
level=error msg="1 error occurred:"
level=error msg="\t* module.vpc.aws_route_table_association.route_net[0]: 1 error occurred:"
level=error msg="\t* aws_route_table_association.route_net.0: timeout while waiting for state to become 'success' (timeout: 5m0s)"
```

**Pull request merged**